ledger/trackerdb.go

```diff
@@ -167,6 +167,13 @@ func trackerDBInitializeImpl(ctx context.Context, tx *sql.Tx, params trackerDBPa
 					tu.log.Warnf("trackerDBInitialize failed to upgrade accounts database (ledger.tracker.sqlite) from schema 5 : %v", err)
 					return
 				}
+				// NOTE wait until 320 round PR merge in master/`feature/avm-box`.
+			case 6, 7:
```

Comment:
accountDBVersion 6 -> 7 is reserved for 320 rounds.

DO NOT MERGE THIS PR UNTIL master MERGED INTO feature/avm-box

Reply:
ok, fixed after 320 in feature/avm-box and updated feature/avm-box merged in